.mockery.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
```

Comment:
This file was authored following the example at https://vektra.github.io/mockery/latest/configuration/#recommended-basic-config and understanding the parameter descriptions at https://vektra.github.io/mockery/latest/configuration/#parameter-descriptions